src/test/java/seedu/eduke8/question/QuizQuestionsManagerTest.java

```diff
@@ -22,6 +22,7 @@
     private static final String PLACEHOLDER_HINT_DESCRIPTION = "Please check the textbook page 88";
```

Comment:
You can load from the test storage `data/test/example.json` if you do not want to create the topic for this test